sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -40,17 +40,27 @@ private[sql] object PythonSQLUtils {
     FunctionRegistry.functionSet.flatMap(f => FunctionRegistry.builtin.lookupFunction(f)).toArray
   }
 
-  def listSQLConfigs(): Array[(String, String, String, String)] = {
+  private def listAllSQLConfigs(): Seq[(String, String, String, String)] = {
     val conf = new SQLConf()
+    // Force to build static SQL configurations
+    StaticSQLConf
+    // set nondeterministic configurations with general meanings
+    conf.getAllDefinedConfs.map {
+      case p @ (SQLConf.SESSION_LOCAL_TIMEZONE.key, _, _, _) =>
+        p.copy(_2 = "value of local timezone")
+      case p @ (StaticSQLConf.WAREHOUSE_PATH.key, _, _, _) =>
+        p.copy(_2 = "value of $PWD/spark-warehouse")
+      case o => o
+    }
```

Comment:
nit: could we move this normalization into the Python script size? I personally think its better that `listRuntimeSQLConfigs`/`listStaticSQLConfigs ` just return the original values. They are only used for documentation now though, I'm not 100% sure about how to use them in the future.

Reply:
Yeah, let's handle it in Python script.